data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -410,7 +410,7 @@ railway-station-subway-tashkent                     # icon z13- (also has captio
 railway-station-subway-tbilisi                      # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tehran                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tianjin                      # icon z13- (also has caption(optional) z13-)
-railway-station-subway-tokyo                        # icon z13- (also has caption(optional) z13-)
+railway-station-subway-tokyo                        # icon z14- and caption z13 (also has caption(optional) z14-)
```

Comment:
I see your `text: none;` addition to the Subways.mapcss, but I see no changes in the generated files (still shows a caption at z13). Did you regenerate them?

Reply:
Yeah I realised that so I put the PR as a draft. Will update with (hopefully) the last needed change shortly.